app/controllers/concerns/publishers/wizardable.rb

```diff
@@ -96,8 +96,16 @@ def important_dates_params(params)
   end
 
   def applying_for_the_job_params(params)
-    params.require(:publishers_job_listing_applying_for_the_job_form)
-          .permit(:application_link, :enable_job_applications, :contact_email, :contact_number, :personal_statement_guidance, :school_visits, :how_to_apply)
+    if params[:publishers_job_listing_applying_for_the_job_form]
```

Comment:
When is `params[:publishers_job_listing_applying_for_the_job_form]` not present?

Reply:
it's not present when you submit the form without selecting any radio button on it. Because it's the only field now on that page, and we know that the govuk_form_builder does not send any forms if all the values are empty, we need that.